daml/Marketplace/Listing.daml

```diff
@@ -6,9 +6,12 @@ type T = Listing
 
 data Status
     = Active
-    | Inactive
+    | Disabled
+    | Failure with
```

Comment:
Don't think that failure should be captured here.
Also I've mentioned before, I would avoid making this specific to exberry.

Reply:
You're correct, this is quite specific to Exberry. My issue is that since this instrument creation with Exberry isn't documented, I don't actually know what's captured in the message vs name field so I left that in to refactor once I test with them. Plus, we need to agree on what to capture generically during failure scenarios.